spec/cucumber/core/event_bus_spec.rb

```diff
@@ -0,0 +1,151 @@
+require "cucumber/core/event_bus"
+
+module Cucumber
+  module Core
+    module Events
+
+      class TestEvent < Core::Event.new(:some_attribute)
+      end
+
+      AnotherTestEvent = Core::Event.new
+
+      UnregisteredEvent = Core::Event.new
+    end
+
+    describe EventBus do
+      let(:bus) { EventBus.new(registry) }
```

Comment:
I'm not very comfortable with using a variable named `events` referring to the `EventBus`, but if it is used in the Cucumber-Ruby-Core code, I think is should be used here to, so this spec describes also that aspect of the use of the `EventBus`, that is "when you see a variable named `events` is most likely is the `EventBus`".